ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Jenkins_User/tasks/main.yml

```diff
@@ -64,6 +64,6 @@
   when:
     - ansible_architecture == "x86_64"
     - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
-    - ansible_distribution_major_version == "6"
+    - ansible_distribution_major_version == "6" or ansible_distribution_major_version == "7"
```

Comment:
Is this definitely needed on 7? If so, why would it have been locked to 6 in the first place?

Reply:
Looking at the original PR it was only tested on 6. This is also only needed on perf machines so may have not been spotted on previous test machines.

Reply:
Where is the documented requirement for that on RHEL7?

Reply:
@smlambert Can you comment on this? core hard limit is unlimited on RHEL7 by default I believe, open files `4096` and process 7260. I can see that some of the performance tests might just about need more than 4096 files open but I'd have thought the other limits wouldn't be a problem with the defaults for performance machines.

Reply:
@smlambert Any view on my previous comment?

Reply:
We did hit an issue with file handle limits for some tests.  I do not recall hitting any other resource issues though.

Reply:
> @sxa can we merge this now?

Do we have the access control on this machine set up, or is there a reason not to? I don't want to merge until the setup of the machines is finalised since they will otherwise be difficult to support.

Reply:
@sxa I've added your key to both machines. The KeyBox key is also on the machines